test/OpenTelemetry.Extensions.Tests/Trace/AutoFlushActivityProcessorTests.cs

```diff
@@ -40,7 +40,7 @@ public void AutoFlushActivityProcessor_FlushAfterLocalServerSideRootSpans_EndMat
 
         using var source = new ActivitySource(sourceName);
         using var activity = source.StartActivity("name", ActivityKind.Server);
-        activity.Dispose();
+        activity.Stop();
```

Comment:
@utpilla FYI this change didn't help, but bumping the version of Moq actually seems to have finally stabilized this test. I looked through the CHANGELOG but nothing jumped out at me as an obvious fix 🤷

Relates to #814